find_me_a_sandwich/app/controllers/favorites_controller.rb

```diff
@@ -0,0 +1,27 @@
+class FavoritesController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+  end
+
+  def friends
+    @friends = User.visible.where.not(id: current_user.id)
+    @friends_favorites, @total_count = Favorite.friends_favorites(@friends)
+  end
+
+  def create
+    Favorite.create(user_id: current_user.id,
```

Comment:
It would be easier to use the `favorites` navigation method on the current user here.  Then you can take advantage of your `favorite_params` method.

Example:

```ruby
current_user.favorites.create(favorite_params)
```